Neos.Neos/Configuration/Settings.DocTools.yaml

```diff
@@ -0,0 +1,195 @@
+Neos:
+  Flow:
+    object:
+      includeClasses:
+        'neos.contentrepositorycore': [ 'Neos\\ContentRepository\\Core\\Feature\\.*' ]
```

Comment:
aam i think we dont want to proxy those classes? As the core library uses of then php that is too modern for flow.

Reply:
Made the needed configuration a comment, we'll add it during rendering in Jenkins…

Reply:
okay thanks i see now what you meant: there is just a comment and via https://github.com/neos/neos-development-distribution/pull/108 we write the settings during rendering. Thanks for the hotfix while this is a little hacky it hopefully troubles us less than proxying anything of the core at runtime :D